README.md

```diff
@@ -3,11 +3,11 @@
 [![ReadTheDocs](https://readthedocs.org/projects/mercury-global-loadbalancer/badge/?version=latest)](http://mercury-global-loadbalancer.readthedocs.io/en/latest/)
 
 # Mercury
-Mercury is a loadbalancer with Global Loadbalance capababilities across multiple Datacentre or Cloud infrastructures.
+Mercury is a load balancer with Global Load balance capabilities across multiple Datacenter or Cloud infrastructures.
```

Comment:
Data centre is British, center is American, I'd personally suggest that this is breaking a correct spelling ;)

But no objections really :)